Makefile.PL

```diff
@@ -0,0 +1,79 @@
+#Makefile.PL version 1.0
+
+use 5.010001;
+use ExtUtils::MakeMaker;
+
+my @scripts   = qw(htseq2multx.pl);
+my $fqmx_link = "https://github.com/brwnj/fastq-multx";
+my $fqmx_name = 'fastq-multx';
+
+WriteMakefile
+  (
+   NAME            => 'htseq2multx',
+
+   #Add these new keywords supported since 5.005
+   ($[ >= 5.005 ?
+
+    (AUTHOR        => 'Robert W. Leach (rleach@princeton.edu)',
+     ABSTRACT_FROM => "$scripts[0]") : ()),
+
+   VERSION_FROM    => "$scripts[0]", # finds $VERSION
+   PREREQ_PM       => {'IO::Pipe::Producer'     => 2.01,
+                       'IO::Select'             => 1.21,
+                       'IO::Uncompress::Gunzip' => 2.060,
```

Comment:
The trailing zero (`0`) requires this be quoted.
```suggestion
                       'IO::Uncompress::Gunzip' => '2.060',
```

Reply:
Yeah, I thought it might.  (I don't know why I'm going through these in reverse order, but as below), I was surprised that the check on my system succeeded without the quotes.  I expected it to fail and that I'd have to add quotes, but it didn't.  I will add the quotes.

- [ ] Add quotes to the Gunzip module

And since we're requiring gunzip anyway, I realize we can use it in the test, so, I will make your suggested gunzip change below.